backend/webserver/models.py

```diff
@@ -15,3 +16,61 @@ class FollowRequest(models.Model):
 class Follow(models.Model):
     follower = models.ForeignKey(Author, related_name='following_authors', on_delete=models.CASCADE)
     followee = models.ForeignKey(Author, related_name='followed_by_authors', on_delete=models.CASCADE)
+
+class Post(models.Model):
+
+    author = models.ForeignKey(Author,on_delete=models.CASCADE)
+
+    created_at = models.DateTimeField(verbose_name="date created",null=False, blank=False, default = datetime.now)
+
+    edited_at = models.DateTimeField("date edited")
+
+    title = models.CharField(max_length=200)
+
+    description = models.CharField(max_length=200)
+
+    source = models.CharField(max_length=200)
+
+    origin = models.CharField(max_length=200)
+
+    unlisted = models.BooleanField(default=False) 
+
+    VISIBILITY_CHOICES = [
+        ("PUBLIC","Public"),
+        ("FRIENDS","Friends"),
+    ]
+
+    visibility = models.CharField(max_length=200,choices=VISIBILITY_CHOICES,default="PUBLIC")
+
+    CONTENT_TYPE_CHOICES = [
+        ("text/plain","Plain text"),
+        ("text/markdown","Markdown text")
+    ]
+
+    content_type = models.CharField(max_length=200,choices=CONTENT_TYPE_CHOICES,null=False,default="text/plain")
+
+    content = models.CharField(max_length=200)
```

Comment:
```suggestion
    content = models.TextField(blank=True)
```
- `TextField` because content could be very large
- `blank=True` because frontend could send us an emtpy string